IndicoV2/Workflows/WorkflowsClient.cs

```diff
@@ -18,7 +18,13 @@ public WorkflowsClient(IndicoClient indicoClient)
             _strawberryShake = indicoClient.IndicoStrawberryShakeClient;
         }
 
-        public async Task<IEnumerable<IWorkflow>> ListAsync(int dataSetId, CancellationToken cancellationToken = default) 
+        public async Task<IWorkflow> GetWorkflowAsync(int workflowId, CancellationToken cancellationToken = default)
+        {
+            var result = await _strawberryShake.Workflows().GetWorkflowAsync(workflowId, cancellationToken);
+            return result.Workflows.Select(x => ToWorkflow(x)).ToList()[0];
```

Comment:
you can do `FirstOrDefault()`, or something similar here. That way if there are no workflows returned, it can return null instead of throwing an exception when you try to access the list with [0]

Reply:
changed!